tests/DevApps/ciam/myWebApi/Program.cs

```diff
@@ -11,19 +11,11 @@
     .AddMicrosoftIdentityWebApi(builder.Configuration.GetSection("AzureAd"));
 
 builder.Services.AddControllers();
-// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+// Learn more about configuring OpenAPI at https://aka.ms/aspnetcore/swashbuckle
```

Comment:
```suggestion
// Learn more about configuring OpenAPI at https://learn.microsoft.com/en-us/aspnet/core/fundamentals/openapi/aspnetcore-openapi?view=aspnetcore-8.0&tabs=visual-studio%2Cminimal-apis
```
**Sugestion:** The currently linked docs seem to be more Swagger-focused. I think this is a better resource, but I could not find an aka.ms link